new-lamassu-admin/src/components/inputs/base/TextInput.js

```diff
@@ -40,14 +81,23 @@ const TextInput = memo(
   }) => {
     const classes = useStyles()
 
+    const error = !!(touched[name] && errors[name])
+
+    const classNames = {
+      [props.className]: true,
+      [classes.filled]: !error && value,
+      [classes.empty]: !value || value === ''
+    }
+
     return (
       <TextField
         id={name}
         onChange={onChange}
         onBlur={onBlur}
-        error={!!(touched[name] && errors[name])}
```

Comment:
We need to remove this formik stuff from here, but it shouldn't break things. Errors should only show if it was already touched, can you add this logic to the formik specific text input?